package-lock.json

```diff
@@ -1,3 +1,11 @@
 {
-  "lockfileVersion": 1
+  "requires": true,
```

Comment:
Doesn't seem like this should be here, or associated node_modules files.

Reply:
You are right, it's a mess, I will try to fix the modules.